target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -160,7 +160,9 @@ ex2700)
 	;;
 f5d8235-v1|\
 f5d8235-v2)
+	ucidef_set_led_default "power" "POWER" "$board:blue:router" "1"
```

Comment:
Just to reconfirm. Dropping this line would mean that the router LED will blink indefinitely no matter in which state the device is.

The change to blue:wired sounds good even though it means that I need to split V1 and V2 again as the blue:wired is missing there as I learned minutes before ;)